### ISSUE TITLE
Missed copyrights for libgcc-s1, libgomp1, libstdc++6 packages

### ISSUE THREAD

**Issue by pavelchavyr** [GoogleContainerTools/distroless#1395]
Checked with gcr.io/distroless/java17-debian11.

Copyrights for all installed packages saved in /usr/share/doc directory.
But the copyrights for the libgcc-s1, libgomp1, libstdc++6 are just symbolic links to the copyright for the gcc-10-base, which is not installed and therefore missing.
![Screenshot from 2023-09-08 18-04-27](https://github.com/GoogleContainerTools/distroless/assets/71088513/819e0f1e-0b5d-4b91-8429-a89de01c1112)

**Comment by loosebazooka** [comment]
Ugh, well that's a weird quirk. I guess we can pull that package and extract the copyright.

**Comment by pavelchavyr** [comment]
Yes, I think it is good idea. But I'm not so familiar with this code. Maybe someone can do that?

**Comment by polarathene** [comment]
Is this a legal requirement to distribute the copyrights within the image content? Or can that be resolved via other means?
